codec-http2/src/main/java/io/netty/handler/codec/http2/Http2StreamChannel.java

```diff
@@ -13,22 +13,21 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
+
 package io.netty.handler.codec.http2;
 
-import io.netty.util.internal.ObjectUtil;
+import io.netty.channel.Channel;
 import io.netty.util.internal.UnstableApi;
 
+// TODO: Should we have an extra method to "open" the stream and so Channel and take care of sending the
+//       Http2HeadersFrame under the hood ?
+// TODO: Should we extend SocketChannel and map input and output state to the stream state ?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)